### DISCUSSION TITLE
MAE와 MSE

### DISCUSSION THREAD

**Discussion by minji-o-j** [Boostcamp-AI-Tech-1-15/ddobokki_log#15] [2. DL Basic]
MSE가 제곱을 하기 때문에 outlier가 너무 커질 수도 있는 문제가 있다고 했는데,
오차가 1 이상일 경우, MSE는 제곱을 하고, MAE는 절댓값을 씌우므로 MSE가 무조건 더 커지는 것 아닌가요??

매번 훈련시 오차가 1 이상일지, 이하일지 알 수 없는 상황에서는 MAE를 쓰는 것이 더 나은 판단 아닌가요?
그럼에도 불구하고 MSE를 손실함수로 더 자주 쓴다고 하던데 이유가 궁금합니다.  아니면 제가 혹시 잘못 알고 있는 건지도 알고싶어요!

**Comment by keemdy** [comment]
MLE는 gradient가 항상 일정해서 최저점 찾기가 어렵고 MSE는 오차가 줄어들면 gradient도 줄어들기 때문에 계산이 수월해서 아닐까요 ?
[https://www.quora.com/What-is-the-difference-between-squared-error-and-absolute-error](url)
참고하면 도움이 될 것 같습니다 ~~!

**Comment by mybirth0407** [comment]
L1 norm, L2 norm 에 대한 이야긴데요, 이는 regression model에서 ridge, lasso 모델과 유사한데요.
https://brunch.co.kr/@itschloe1/11
이 링크를 참고하시면 좋을 것 같습니다.
